user/languages/rust.md

```diff
@@ -86,6 +86,19 @@ cache: cargo
 ```
 {: data-file=".travis.yml"}
 
+This adds the following directories to the cache:
+
+- `$TRAVIS_HOME/.cache/sccache`
+- `$TRAVIS_HOME/.cargo/`
+- `$TRAVIS_HOME/.rustup/`
+- `target`
+
+In addition, it adds the following command to the `before_cache`
+phase of the job in order to reduce cache size:
+
+    rm -rf "$TRAVIS_HOME/.cargo/registry/src"
+
+If you override `before_cache`, we recommend adding the above.
```

Comment:
```suggestion
If you override the `before_cache` step, we recommend adding the  step above: 

```yaml
before_cache: 
   - rm -rf "$TRAVIS_HOME/.cargo/registry/src"
```

```

Not sure if suggestions + code comments would apply properly

Reply:
How about something like:

> This means that, if you override the `before_cache` step for another reason, you should add the step above in order to reduce the cache size:
> 
> ```yaml
> before_cache:
>   - rm -rf "$TRAVIS_HOME/.cargo/registry/src"
>   ⋮ # rest
> ```

Reply:
perfect! thanks @BanzaiMan